src/importIssues.ts

```diff
@@ -189,10 +188,12 @@ export const importIssues = async (apiKey: string, importer: Importer) => {
       name: 'targetAssignee',
       message: 'Assign to user:',
       choices: () => {
-        return users.map((user: { id: string; name: string }) => ({
+        const map = users.map((user: { id: string; name: string }) => ({
           name: user.name,
           value: user.id,
         }));
+        map.push({ name: '[[Unassigned]]', value: '' });
```

Comment:
```suggestion
        map.push({ name: '[Unassigned]', value: null });
```

Little nitpicky but `null` would be better to indicate that there won't be a value. It might even work for the graphql query as is?

Reply:
so `null` won't work because ImportAnswers requires a `string`